user-docs/pupil-player.md

```diff
@@ -149,15 +149,58 @@ This plugin can be used to overlay the eye video on top of the world video. Note
   + `show` - show or hide eye video overlays.
   + `horiz. and vert. flip` - flip eye videos vertically or horizontally
 
-### Analysis Plugins
+### Data Source Plugins
 
-These plugins are simple unique plugins, that operate on the gaze data for analysis and visualizations.
+Data source plugins provide data of a specific topic -- currently `pupil` and `gaze` positions. Plugins that provide data for a specific topic are complementary to each other, e.g. `Pupil Positions From Recording` and `Offline Pupil Detector` (see below). Complementary plugins cannot be run at the same time.
+
+#### Pupil Positions From Recording
+
+This plugin tries to load the pupil positions that were detected and stored during a Pupil Capture recording. It is loaded by default and is complementary to the `Offline Pupil Detector`.
+
+#### Offline Pupil Detector
+Starting with version `v9.13`, Pupil Player is able to run the same pupil detection algorithm as in Pupl Capture for recorded eye videos. This is especially important for Pupil Mobile recordings, since the app does not run pupil detection on the phone. The plugin tries to load the `eye0.mp4` and `eye1.mp4` videos in the current recording and runs the pupil detection in a separate process. `Offline Pupil Detector` is complementary to `Pupil Positions From Recording`.
```

Comment:
Again, complementary is (IMO) a misleading word choice here. Here is an attempt to re-write. 

The `Offline Pupil Detector` plugin can be used on any dataset where eye videos were recorded. The plugin tries to load the `eye0.mp4` and `eye1.mp4` videos, and runs pupil detection algorithm in separate processes. This plugin is especially relevant for recordings made with Pupil Mobile, because Pupil Mobile does not perform any pupil detection or gaze estimation on the Android device. This plugin is available starting with Pupil Player `v0.9.13`.